pkg/ddevapp/config.go

```diff
@@ -1481,6 +1481,36 @@ func AvailablePHPDocrootLocations() []string {
 	}
 }
 
+// CreateDocroot normalizes the docroot path and creates it for DDEV app if it doesn't exist
+func (app *DdevApp) CreateDocroot() error {
+	if app.Docroot == "" {
+		return nil
+	}
+	docrootAbsPath := ""
+	if filepath.IsAbs(app.Docroot) {
```

Comment:
This works correctly, but it's not really clear to the user that the absolute path is a host-side docroot.  No need to change, but I would have preferred simpler approach that did not allow absolute path here. I think it adds complexity and code, but doesn't offer much to the user.